MANIFEST.in

```diff
@@ -1,3 +1,6 @@
-include curl_cffi/_wrapper.*
-include curl_cffi/ffi/*
-include curl_cffi/include/curl/*
+recursive-include tests/*
+
+include ffi/*
+include preprocess/*
+include scripts/build.py
+include Makefile
```

Comment:
Do we need to also include .jpg files (as dependency of README file) and translated `README-zh.md`?

Reply:
They are hard-linked to github, SoI think it's not required.